ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -84,6 +84,13 @@ export const ApplicationOperationState: React.StatelessComponent<Props> = ({appl
         }
     }
 
+    const [filters, setFilters] = React.useState([]);
+    const filterOptions = ['Synced', 'OutOfSync', 'Unknown'];
+
+    let filtered = syncResult.resources;
```

Comment:
Tried to create a brand new app, sync it, and reproduced a Javascript error here. `syncResult` might be undefined.